packages/path_provider/path_provider_android/test/path_provider_android_test.dart

```diff
@@ -52,14 +52,6 @@ void main() {
       log.clear();
     });
 
-    // TODO(stuartmorgan): Change this to a test that it uses a different name,
```

Comment:
I removed it rather than changing it, since the reverse test didn't seem to actually be useful (and we don't have it anywhere else). I could add it if you think it's worthwhile though.